src/Frontend/Fragment/Feasibility.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE RankNTypes          #-}
+
+{-# OPTIONS_GHC -Werror -Wall #-}
+
+-- | This module contains snippets which are used all over the pages.
```

Comment:
is this comment intentional, or a left-over?  if the former, please keep it!  (-: